packages/wallet/backend/src/account/service.ts

```diff
@@ -295,4 +308,84 @@ export class AccountService implements IAccountService {
 
     return account
   }
+
+  public async createExchangeQuote({
```

Comment:
as a lot of the logic repeats for sender and receiver, I suggest to try to extract the logic that can be reused for both in multiple small methods